final-project/src/tier1/action_primitives.py

```diff
@@ -33,7 +33,7 @@ def straight_line_rotations(self):
         matrix[0] = np.matrix(
             [[0.0, 0.0],
             [0.0, 0.0],
-            [0.0, 1.0]]
```

Comment:
Why change this to 1.425?

Reply:
The resulting point wasn't reaching 1.0, so I made it greater than 1 or less then -1, so I could clip at 1 or -1.

Reply:
I think we need a comment for that since it is very context dependent.